docs/asciidoc/faq.adoc

```diff
@@ -102,6 +102,25 @@ logger (`_reactor.netty.channel.LeakDetection`) that helps to identify where the
 or whether Reactor Netty already forwarded the ownership of the buffers to the application/framework.
 By default, this logger is disabled. To enable it, increase the log level to `DEBUG`.
 
+Another way to detect native memory leaks is to use _PooledByteBufAllocator_ metrics.
+You can obtain the _PooledByteBufAllocatorMetric_ returned by a PooledByteBufAllocator in order to monitor the currently allocated direct buffers
+(but it won't help to figure out from where the memory leak is happening). Among other things, the  _PooledByteBufAllocatorMetric_ provides the list of
+direct arenas that are used for *direct memory* buffer pooling, and you can then periodically sum direct buffer allocatations from all direct arenas, like in the following example:
+
+```
+        PooledByteBufAllocator alloc = PooledByteBufAllocator.DEFAULT;
+        PooledByteBufAllocatorMetric metrics = alloc.metric();
+
+        // display the total number of buffers allocated from direct memory arenas:
+        while (true) {
+             Thread.sleep(1000);
+             System.out.println(metrics.directArenas().stream()
+                 .collect(Collectors.summingLong(PoolArenaMetric::numAllocations)));
+        }
+```
+
+If the above code dislays some growing values, it's likely that there is a direct buffer memory leak.
+
```

Comment:
May be it is better to extend our Memory metrics. Wdyt?

Reply:
This discussion is interesting https://github.com/netty/netty/pull/11642

Reply:
yes !, better is to add more meter(s).
ok, I'll work on this ...